board/safety_declarations.h

```diff
@@ -220,6 +223,9 @@ uint32_t ts_angle_last = 0;
 int desired_angle_last = 0;
 struct sample_t angle_meas;         // last 6 steer angles/curvatures
 
+// for safety modes that need to passthru aeb
+bool stock_aeb = false;
+
```

Comment:
I think we can do AEB safety and passthrough in a separate PR. We make no guarantees for any alpha longitudinal behind the debug flag

Reply:
I'll remove aeb stuff for now and move it to a new PR